src/util.py

```diff
@@ -260,12 +265,103 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+def set_vector_density(data, lat_density=1, lon_density=1, minDistance=0, otherVars=[]):
+    """
+    Utility function to change density of vector plots.
+
+    Ars:
+
+        data (:class:`xarray.core.dataarray.DataArray`):
+            Data array that contains the vector plot data.
+
+        lat_density (:class:`int`):
+            Value in range (0,1] that determines the density of the vectors in the y range.
+
+        lon_density (:class:`int`):
+            Value in range (0,1] that determines the density of the vectors in the x range.
+
+        minDistance (:class:`int`):
```

Comment:
minDistance is what was asked for in the issue because it is most similar to NCL's vcMinDistanceF. I added on lat_density and lon_density as extra arguments, which will be useful if the user wants to have a different density of vectors in the x versus the y direction.

Reply:
Ah I wasn't aware. Including it is a good idea if that's the case.